### PR TITLE
Update: new xerial api connection

### PR DESCRIPTION
**Pull request by RichardIrala**

### CODE REVIEW

**Comment by RichardIrala** [comment]
* Update: new xerial api connection
* Fix: list and delist nft. Update: toastNotify added into UI. New dependency added: react-toastify
* Update: MATIC/TLOS Currencies added into UI

File: atoms/InventoryItemOnSecondaryMarket/InventoryItemOnSecondaryMarket.jsx [hunk 1]
```diff
@@ -1,8 +1,15 @@
...
+import toastNotify from "@/utils/toastNotify";
 
 const InventoryItemOnSecondaryMarket = ({ nft, activeChain }) => {
   async function onDelistNft() {
-    await web3Functions.delistNftOnSecondaryMarket({ marketplaceNftId: nft.marketItemId, chain: activeChain });
+    try {
+      await web3Functions.delistNftOnSecondaryMarket({ marketplaceNftId: nft.marketItemId, chain: activeChain });
+    } catch (error) {
+      console.error(error.message);
+      toastNotify("Error: Delist NFT Failed", "error");
```
Comment (luislucena16) [inline atoms/InventoryItemOnSecondaryMarket/InventoryItemOnSecondaryMarket.jsx:11]: maybe here you can use:
```javascript
toastNotify("Error: Delist NFT Failed - " + error);`
```

File: pages/index.js [hunk 2]
```diff
@@ -210,12 +210,12 @@ export default function Home({ XerialWalletViewmodel, activeChain, handleActiveC
...
             </div>
             <div className="home__projectBannerContainer">
-              <img className="home__projectBanner" src={project.userBanner} />
+              <img className="home__projectBanner" src={project.banner} />
             </div>
             <div className="home__projectDataSubcontainer">
               <div className="home__projectData">ID: {project.id}</div>
               <div className="home__projectData">Name: {project.name}</div>
-              <div className="home__projectData">Description: {project.description}</div>
+              {/* <div className="home__projectData">Description: {project.description}</div> */}
```
Comment (luislucena16) [inline pages/index.js:218]: any reason not to delete this?

File: utils/toastNotify.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import { toast } from "react-toastify";
+
+// // Allows you to add notifications in UI
+// // type posible values: info, success, warning, error
```
Comment (luislucena16) [inline utils/toastNotify.js]: improve this:
```javascript
// Allows you to add notifications in UI
// type posible values: info, success, warning, error
```

File: viewmodels/xerialWallet.js [hunk 4]
```diff
@@ -21,8 +21,8 @@ export default function XerialWalletViewmodel(helpers) {
     const result = await helpers.getProjectForMarketplace({
       projectDomain: projectDomain
     });
-    if (!result.project) throw new Error("Project not found.")
-    vm.setProject(result.project);
+    if (!result) throw new Error("Project not found.")
```
Comment (luislucena16) [inline viewmodels/xerialWallet.js]: use: `Project Not Found`

**Review by luislucena16 — CHANGES_REQUESTED** [review]

File: atoms/InventoryItemOnSecondaryMarket/InventoryItemOnSecondaryMarket.jsx (hunk 1 above)
Comment (RichardIrala) [reply atoms/InventoryItemOnSecondaryMarket/InventoryItemOnSecondaryMarket.jsx:11]: toastNotify("Error: Delist NFT Failed", "error"); it's okay

The second param indicates the notify type

File: pages/index.js (hunk 2 above)
Comment (RichardIrala) [reply pages/index.js:218]: I didn't know if the removal of the description is temporary or permanent on the backend

File: utils/toastNotify.js (hunk 3 above)
Comment (RichardIrala) [reply utils/toastNotify.js]: Sure

Comment (RichardIrala) [reply utils/toastNotify.js]: My bad

File: viewmodels/xerialWallet.js (hunk 4 above)
Comment (RichardIrala) [reply viewmodels/xerialWallet.js]: Done!

File: atoms/InventoryItemOnSecondaryMarket/InventoryItemOnSecondaryMarket.jsx (hunk 1 above)
Comment (luislucena16) [reply atoms/InventoryItemOnSecondaryMarket/InventoryItemOnSecondaryMarket.jsx:11]: yes, either way it is valid, I think we were using it before with the `+`

File: pages/index.js (hunk 2 above)
Comment (luislucena16) [reply pages/index.js:218]: ok I understand!

**Review by luislucena16 — APPROVED** [review]

**Pull request merged**